components/aws/sagemaker/ground_truth/src/sagemaker_ground_truth_component.py

```diff
@@ -0,0 +1,348 @@
+"""SageMaker component for ground truth."""
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+from typing import Dict
+
+from ground_truth.src.sagemaker_ground_truth_spec import (
+    SageMakerGroundTruthSpec,
+    SageMakerGroundTruthInputs,
+    SageMakerGroundTruthOutputs,
+)
+from common.sagemaker_component import (
+    SageMakerComponent,
+    ComponentMetadata,
+    SageMakerJobStatus,
+)
+
+
+@ComponentMetadata(
+    name="SageMaker - Ground Truth",
+    description="Ground Truth Jobs in SageMaker",
+    spec=SageMakerGroundTruthSpec,
```

Comment:
Why is this needed ?

Reply:
Specifying the spec in the decorator allows the component compiler to find the inputs and outputs.